code/datums/traits/negative.dm

```diff
@@ -448,3 +448,176 @@ GLOBAL_LIST_EMPTY(family_heirlooms)
+			to_chat(quirk_holder, "<span class='danger'>You thought you kicked it, but you suddenly feel like you need [reagent_instance.name] again...")
+		tick_counter = 0
+	else
+		++tick_counter
+
+/*
+/datum/quirk/junkie/smoker
+	name = "Smoker"
+	desc = "Sometimes you just really want a smoke. Probably not great for your lungs."
+	value = -1
+	gain_text = "<span class='danger'>You could really go for a smoke right about now.</span>"
+	lose_text = "<span class='notice'>You feel like you should quit smoking.</span>"
+	medical_record_text = "Patient is a current smoker."
+	reagent_type = /datum/reagent/drug/nicotine
+	accessory_type = /obj/item/lighter/greyscale
+/datum/quirk/junkie/smoker/on_spawn()
+	drug_container_type = pick(/obj/item/storage/fancy/cigarettes,
+		/obj/item/storage/fancy/cigarettes/cigpack_midori,
+		/obj/item/storage/fancy/cigarettes/cigpack_uplift,
+		/obj/item/storage/fancy/cigarettes/cigpack_robust,
+		/obj/item/storage/fancy/cigarettes/cigpack_robustgold,
+		/obj/item/storage/fancy/cigarettes/cigpack_carp,
+		/obj/item/storage/fancy/cigarettes/cigars,
+		/obj/item/storage/fancy/cigarettes/cigars/cohiba,
+		/obj/item/storage/fancy/cigarettes/cigars/havana)
+	. = ..()
+/datum/quirk/junkie/smoker/announce_drugs()
+	to_chat(quirk_holder, "<span class='boldnotice'>There is a [drug_instance.name] [where_drug], and a lighter [where_accessory]. Make sure you get your favorite brand when you run out.</span>")
+/datum/quirk/junkie/smoker/on_process()
+	. = ..()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/obj/item/I = H.get_item_by_slot(SLOT_WEAR_MASK)
+	if (istype(I, /obj/item/clothing/mask/cigarette))
+		var/obj/item/storage/fancy/cigarettes/C = drug_instance
+		if(istype(I, C.spawn_type))
+			SEND_SIGNAL(quirk_holder, COMSIG_CLEAR_MOOD_EVENT, "wrong_cigs")
+			return
+		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "wrong_cigs")
+*/
+
+/datum/quirk/junkie/jet
+	name = "Jet Addict"
+	desc = "The jet will make you jittery!"
+	value = -2
+	gain_text = "<span class='danger'>You begin craving a whiff from a jet inhaler.</span>"
+	lose_text = "<span class='notice'>You regret your life decisions and lose interest in jet.</span>"
+	medical_record_text = "Patient is a jet addict."
+	reagent_type = /datum/reagent/drug/jet
+	drug_container_type = /obj/item/reagent_containers/pill/patch/jet
+
+/datum/quirk/junkie/psycho
+	name = "Psycho Addict"
+	desc = "KILL! DEATH! DESTROY!"
+	value = -2
+	gain_text = "<span class='danger'>You begin craving a shot of psycho.</span>"
+	lose_text = "<span class='notice'>You regret your life decisions and lose interest in psycho.</span>"
+	medical_record_text = "Patient is a psycho addict."
+	reagent_type = /datum/reagent/drug/psycho
+	drug_container_type = /obj/item/reagent_containers/hypospray/medipen/psycho
+
+/datum/quirk/junkie/turbo
+	name = "Turbo Addict"
+	desc = "Okay, here we go. Focus. Speed. I am speed. One winner, 42 losers. I eat losers for breakfast. Kachow!"
```

Comment:
One critique. This line seems particularly strange. Is there something else you can think of that's more fallout appropriate?

Reply:
NO, I WANT A CARS REFERENCE IN THE QUIRK DESCRIPTION. DON'T MAKE ME CHANGE IT!!!!! Kachow.

Reply:
@hecksadecimal I can't find a single quote referencing turbo, so I just stuck a jet one in there. They are basically the same thing anyway.